pyfmt/__init__.py

```diff
@@ -1,5 +1,6 @@
 import os
 import shlex
+from shutil import get_terminal_size
```

Comment:
looks like you need to run pyfmt to sort these imports 😛

Reply:
haha how ironic

Reply:
sec... i'll push an update

Reply:
hey never mind, this was right!

Reply:
i thought `from` imports should be below `import` statements.. weird!